### PR TITLE
update client E2E xunit tests

### PR DESCRIPTION
**Pull request by ElizabethOkerio**
### Issues

*This pull request fixes issue #xxx.*

### Description

*Briefly describe the changes of this pull request.*

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: test/EndToEndTests/Tests/Client/Build.Desktop/ActionOverloadingTests/ActionOverloadingEndToEndTests.cs [hunk 1]
```diff
@@ -76,7 +76,7 @@ public void ExecuteActionImport()
         }
 #endif
 
-        [Fact]
+        [Fact(Skip ="TODOa")]
```
Comment (xuzhg) [inline test/EndToEndTests/Tests/Client/Build.Desktop/ActionOverloadingTests/ActionOverloadingEndToEndTests.cs:79]: why skip it? and If have to skip it, please add the detail description?

File: test/EndToEndTests/Tests/Client/Build.Desktop/ClientWithoutTypeResolverTests/OpenTypesWithoutTypeResolverTests.cs [hunk 2]
```diff
@@ -106,7 +106,7 @@ public void ProjectionQuery()
             var results = query.ToList();
         }
 
-        [Fact]
+        [Fact(Skip ="TODO")]
```
Comment (xuzhg) [inline test/EndToEndTests/Tests/Client/Build.Desktop/ClientWithoutTypeResolverTests/OpenTypesWithoutTypeResolverTests.cs:109]: same here?

File: test/EndToEndTests/Tests/Client/Build.Desktop/Microsoft.Test.OData.Tests.Client.csproj [hunk 3]
```diff
@@ -217,6 +220,10 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (xuzhg) [inline test/EndToEndTests/Tests/Client/Build.Desktop/Microsoft.Test.OData.Tests.Client.csproj:223]: remove this?

**Review by gathogojr — APPROVED** [review]
LGTM

**Review by habbes — APPROVED** [review]

**Pull request merged**